custom_dc/custom_data.md

```diff
@@ -50,45 +50,96 @@ Statistical variables must follow a certain model; in particular, they must repr
+To illustrate the difference between variable-per-column and variable-per-row schemas, let's use the schools example data again. In variable-per-column, you would represent the dataset as follows:
 
-With the variable-per-column scheme, data is provided in this format, in this exact sequence:
+**Variable-per-column schema**
+
+| CITY | YEAR | COUNT_PUBLIC_ELEMENTARY | COUNT_PUBLIC_MIDDLE | COUNT_PUBLIC_SECONDARY | COUNT_PRIVATE_ELEMENTARY | COUNT_PRIVATE_MIDDLE | COUNT_PRIVATE_SECONDARY |
+|------|------|-------------------------|---------------------|------------------------|---------------------------|---------------------|-------------------------|
+| San Francisco | 2023 | 300 | 300 | 200 | 100 | 100 | 50 |
+| San Jose | 2023 | 400 | 400 | 300 | 200 | 200 | 100 |
+
+The names that appear in the columns and rows don't need to be DCIDs or follow any convention, because the columns must always be specified in this exact sequence:
 
 _ENTITY, OBSERVATION_DATE, STATISTICAL_VARIABLE1, STATISTICAL_VARIABLE2, …_
 
-There are two columns, the _ENTITY_ and the _OBSERVATION_DATE_, that specify the place and time of the observation; all other columns must be expressed as variables, as described above. To continue with the above example, a CSV file would need to look like this:
+In variable-per-row, the same dataset would be provided as follows:
 
-```csv
-city,year,CountPublicElementary,CountPublicMiddle,CountPublicSecondary,CountPrivateElementary,CountPrivateMiddle,CountPrivateSecondary
-San Francisco,2023,300,300,200,100,100,50
-San Jose,2023,400,400,300,200,200,100
-```
+**Variable-per-row schema**
+
+| CITY | YEAR |  VARIABLE | OBSERVATION |
+|------|------|-----------|-------|
+| San Francisco | 2023 | CountPublicElementary | 300 |
+| San Francisco | 2023 | CountPublicMiddle | 300 |
+| San Francisco | 2023 | CountPublicSecondary | 200 |
+| San Francisco | 2023 | CountPrivateElementary | 100 |
+| San Francisco | 2023 | CountPrivateMiddle | 100 |
+| San Francisco | 2023 | CountPrivateSecondary | 50 |
+| San Jose | 2023 | CountPublicElementary | 400 |
+| San Jose | 2023 | CountPublicMiddle | 400 |
+| San Jose | 2023 | CountPublicSecondary | 300 |
+| San Jose | 2023 | CountPrivateElementary | 200 |
+| San Jose | 2023 | CountPrivateMiddle | 200 |
+| San Jose | 2023 | CountPrivateSecondary | 100 |
+
+The names and order of the columns aren't important, as you can map them to the expected columns in the JSON file. However, the city and variable names must be existing DCIDs. If such DCIDs don't already exist in the base Data Commons, you must provide definitions of them in MCF files.
```

Comment:
'CITY' column in the table above does not have dcids even though we say `However, the city and variable names must be existing DCIDs.`. Should we just say variable column since city names are resolved?

Reply:
That is a good point!  This was just meant to be kind of hypothetical, but it would be clearer to use DCIDs there. Thanks for the catch.